common/model/bundle.c

```diff
@@ -12,6 +12,59 @@
 
 static UT_icd bundle_transactions_icd = {sizeof(iota_transaction_t), 0, 0, 0};
 
+static retcode_t validate_signatures(bundle_transactions_t const *const bundle,
```

Comment:
Could you add inline or method level documentation to this method please?

Reply:
not `inline`, just was referring to documentation inside the method where it makes sense